brownie/network/account.py

```diff
@@ -388,13 +410,13 @@ def __eq__(self, other):
             return other.address == self.address
         return super().__eq__(other)
 
-    def balance(self):
+    def balance(self) -> Wei:
         '''Returns the current balance at the address, in wei.'''
         balance = web3.eth.getBalance(self.address)
         return Wei(balance)
 
 
-def _raise_or_return_tx(exc):
+def _raise_or_return_tx(exc: Any) -> Any:
```

Comment:
pretty sure `exc` will only ever be a `ValueError`

Reply:
That seems to check out. Done.